framework/synapse-framework-exception/src/main/java/io/americanexpress/synapse/framework/exception/model/ErrorCode.java

```diff
@@ -66,7 +66,12 @@ public enum ErrorCode {
     /**
      * Used for when the consumer makes a request to a locked resource.
      */
-    LOCKED(HttpStatus.LOCKED, "Locked.");
+    LOCKED(HttpStatus.LOCKED, "Locked."),
+
+    /**
+     * Used for when the consumer makes a request with change in the Entity State.
+     */
+    CONFLICT(HttpStatus.CONFLICT, "Entity Conflict.");
```

Comment:
I would change message for 'Resource out of sync' instead of 'Entity Conflict', even though this is just temporary code but still.